consensus/src/aggregation/engine.rs

```diff
@@ -666,12 +668,6 @@ impl<
         });
     }
 
-    // Sets the rebroadcast deadline for the given `index`.
-    fn set_rebroadcast_deadline(&mut self, index: Index) {
```

Comment:
Now that we rebroadcast at various deadlines, I found it clearer to remove this helper.